core/iwasm/interpreter/wasm_interp_fast.c

```diff
@@ -5647,13 +5651,34 @@ wasm_interp_call_func_bytecode(WASMModuleInstance *module,
                 goto call_func_from_entry;
             }
 #if WASM_ENABLE_SIMD != 0
+/* TODO: Add x86 with additional #if */
+#define SIMD_V128_TO_SIMDE_V128(v) \
+    vreinterpretq_s32_u8(vld1q_u8((uint8_t *)&(v)))
+
+#define SIMDE_V128_TO_SIMD_V128(sv, v)              \
+    do {                                            \
+        uint8x16_t temp = vreinterpretq_u8_s32(sv); \
+        vst1q_u8((uint8_t *)&(v), temp);            \
+    } while (0)
```

Comment:
I think this is fine for now - I think they're implemented in an agnostic way in the simde header but it's probably better to add other implementations in here too.